metron-interface/metron-config/scripts/package.json

```diff
@@ -9,7 +9,8 @@
     "http-proxy-middleware": "0.17.4",
     "optimist": "0.6.1",
     "serve-favicon": "2.4.2",
```

Comment:
I believe the dependencies csv is used by verify_licenses.sh script to verify Maven dependencies have been properly researched for license compatibility.  Adding these to that file won't do anything because package.json is used by NPM, not Maven.

The licenses are compatible as far as I know.  Is there something else we need to do?